Automattic/jetpack
# Mirroring: Add a standard .github to all mirrors

#### Changes proposed in this Pull Request:

All the mirror repos will want a standard .github that contains a
workflow that closes any PRs. Have the mirroring script automatically
create it.

#### Jetpack product discussion


I think we mentioned it in a team meeting a few weeks ago.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Merge this.
* Try to submit a PR against one of the mirror repos, to the feature/reorg branch. It should be automatically closed.

#### Proposed changelog entry for your changes:


* None needed.
